hw_diag/views/diagnostics.py

```diff
@@ -24,7 +28,16 @@
 
 def read_diagnostics_file():
     diagnostics = {}
+
     try:
+        # Re-generate the cache if it is older than cache_cutoff
```

Comment:
Why do we use this rather than using a library such as Flask-Cache which can cache in memory and is only 1 line of code (decorating the function).

Reply:
Good idea. Yeah, that's cleaner.